tools/hmac/calculateHmacPayments.js

```diff
@@ -0,0 +1,73 @@
+// script to calculate the HMAC signature of Payments webhooks (where the signature is calculated considering
+// a subset of the fields in the payload - i.e. NotificationRequestItem object)
+//
+// Run with: `node calculateHmacPayments.js {hmacKey} {path to JSON file}
+//
+// cd tools/hmac
+// node calculateHmacPayments.js 11223344D785FBAE710E7F943F307971BB61B21281C98C9129B3D4018A57B2EB payload.json
+
+const fs = require('fs');
+const { hmacValidator } = require('@adyen/api-library');
+
+// Ensure correct arguments
+if (process.argv.length !== 4) {
```

Comment:
check if argv length should be 3? Or 4? @gcatanese

Reply:
That's correct